src/Probabilities.h

```diff
@@ -149,6 +149,7 @@ enum ProbName {
 
 	//for function attributes
 	pFuncAttrInline,
+	pFuncAttrAligned,
```

Comment:
Maybe I would recommend a smaller probability. You don't want to have many alignments. That can significantly grow an object file.

Reply:
I will set probability as 25?

Reply:
Let's try it ;)

Reply:
Is this patch ready to merge? I am testing this patch on gcc-10 of GCC Compiler Farm and then merge into csmith-gcc.

Is this fine?

Reply:
No, I would like to squash the 2 commit that you have in this SR into one. Then I'll make another review.

Reply:
Done